content/sips/sip-161.md

```diff
@@ -48,72 +50,155 @@ On the topic of scaling and decentralizing the Synthetix Protocol, the onboardin
 
 The Core Contributor Committee ratification has been formed from a variety of considerations
 
-_Voting System_ the CCC members are voted in by the Core Contributors as they are the ones who are directly influenced by the decisions that are being made, the Core Contributors are also selected because they are the ones who have the most context making the relationship between the Core Contributors and CCC members symbiotic and accountable.
+_Voting System_
+
+The CCC members are voted in by the Core Contributors as they are the ones who are directly influenced by the decisions that are being made, the Core Contributors are also selected because they are the ones who have the most context making the relationship between the Core Contributors and CCC members symbiotic and accountable.
+
+_Core Contributor Engagement_
+
+The Core Contributor Committee is in the best position to understand which resources are lacking within the Core Contributors and the budget capacity while the existing Core Contributors themselves understand which candidates would be most suitable for the role they are applying for, this vouching system ensures that both parties are responsible for what they know best. This also applies for the disengagement of Core Contributors, since this Committee will be regularly interfacing with the CC's they will have the best understanding of performance.
+
+_Core Contributor Compensation_
+
+The CCC have full discretion over the budget provided by the Treasury Council so they are able to ensure that the Core Contributors incentives are aligned with the Synthetix Protocol vision.
+
+_Core Contributor Vision and Prioritization_
+
+Since the Spartan Council is voted on by the Synthetix token holders which means they are positioned to serve in the best interest of the token holders, there needs to be a mediator between the Spartan Council and the Core Contributors. By position the CCC as this mediator, the Spartan Council can have more transparency and discretion over the performance and prioritization of the Core Contributors ensuring they serve in the best interest of the token holders.
+
+_CCC & CC Stipends_
+
+The CCC Stipends are made transparent so that the CC's have some sort of benchmark in understanding how the CCC are valuing their involvement with the Synthetix Protocol, since the CC Stipends are not public because of any potential conflict that may occur between CC's.
+
+#### Technical Specification
+
+- Internal Infrastructure
+
+  - Work stream for vouching for candidates
+  - Platform for voting on eligible candidates and record keeping on decision making
+  - Platform for voicing opinions on prioritization (Canny)
+  - Communication medium, such as a dedicated Discord channel or forum
+
+- Committee Election Platform
```

Comment:
Maybe add a sentence that this would work similarly to how elections are held today on the staking dapp under governance?